analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AvoidDateTimeNowForBenchmarking.vb

```diff
@@ -0,0 +1,79 @@
+﻿Imports System
```

Comment:
Please add a test thate validates that the string comparisons for  VB are good (case insensitive).

Reply:
You could also add a test case for the scenario with the operator -.
You only covered the method invocation.